lib/build.js

```diff
@@ -258,6 +263,13 @@ export default {
         env[key] = replace(env[key], target.env);
       });
 
+      if (variant) {
+        const copy = {};
+        Object.assign(copy, target);
+        Object.assign(copy, target.variants[variant]);
+        target = copy;
```

Comment:
Since these properties override whatever is set in the target, we make a copy so that building, then testing, then building doesn’t test instead.